src/device.js

```diff
@@ -1,11 +1,13 @@
+import { ADB } from './adb';
+
 /**
  * Detects connected devices.
  *
- * @param {DataType} variablename - description
+ * @param {Object} [opts] - Detection options.
  * @returns {Promise}
  */
-export function detect() {
-}
-
-export function install() {
```

Comment:
Add an explicit `install()` method to wrap `ADB.installApp()` so that andoridlib is in parity with ioslib and windowslib.